content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,183 @@
+click the :guilabel:`Journal Items` tab to view (or change) the :guilabel:`Account` journals that
+were populated based on the configuration on the :guilabel:`Vendor` and :guilabel:`Product` forms.
 
-How product quantities are managed
-----------------------------------
+Then, click :guilabel:`Add Credit Note` or :guilabel:`Add Debit Note` to add credit or debit notes
+to the bill; add a :guilabel:`Bill Reference` number; then click :menuselection:`Register Payment
+--> Create Payment` to complete the :guilabel:`Vendor Bill`.
 
-By default, services are managed based on ordered quantities, while
-stockables and consumables are managed based on received quantities.
+.. tip::
+   To tie the :guilabel:`Draft Bill` to an existing purchase order, click the :guilabel:`drop-down`
+   menu next to :guilabel:`Auto-Complete`, and select a :abbr:`PO (Purchase Order)` from the menu.
+   The bill will auto-populate with the information from the :abbr:`PO (Purchase Order)`.
 
-If you need to manage products based on ordered quantities over received
-quantities, you will need to belong to the group **Purchase Manager**.
-Ask your system administrator to enable these access on :menuselection:`Settings
---> Users --> Users --> Access Rights`. Once you belong to the correct group,
-select the product(s) you wish to modify, and you should see a new field appear,
-labeled **Control Purchase Bills**.
+.. image:: manage/manage-auto-complete.png
+   :align: center
+   :alt: Auto-complete drop-down list on draft vendor bill.
 
-.. image:: manage/manage08.png
-  :align: center
+Batch billing
+=============
 
-You can then change the default management method for the selected
-product to be based on either:
+Vendor bills can be processed and managed in batches in the :guilabel:`Accounting` app. To do this,
+go to :menuselection:`Accounting --> Vendors --> Bills`. Then, click the :guilabel:`checkbox` at
+the top left of the page under :guilabel:`Create`. This will select all existing vendor bills with
+a :guilabel:`Posted` or :guilabel:`Draft` :guilabel:`Status`.
 
-- Ordered quantities
+From here, click the :guilabel:`Action` gear icon to export, delete, or send & print the bills;
+click the :guilabel:`Print` icon to print the invoices or bills; or click
+:guilabel:`Register Payment` to create and process payments for multiple vendor bills at once.
+Select the appropriate :guilabel:`Journal`, choose a :guilabel:`Payment Date` and
+:guilabel:`Payment Method`, and click :guilabel:`Create Payment`. This will create a list of
+journal entries all tied to their appropriate vendor bills.
 
-- or Received quantities
+.. image:: manage/manage-batch-billing.png
+   :align: center
+   :alt: Batch billing register payment popup.
 
-Batch Billing
--------------
+.. note::
+   Clicking :guilabel:`Register Payment` to create payments for vendor bills in batches will only
+   work for journal entries whose :guilabel:`Status` is set to :guilabel:`Posted`.
```

Comment:
The :guilabel:`Register Payment` option for vendor bills in batches will only work for journal entries 
   with a :guilabel:`Status` set to :guilabel:`Posted`.